docs/rest_api.md

```diff
@@ -967,7 +984,9 @@ Response body:
         "name": <headnode>,
         "project": <projectname>,
         "nics": [<nic1>, <nic2>, ...],
-        "vncport": <port number>
+        "vncport": <port number>,
+        "uuid": <headnode uuid>,
+        "base_img": <headnode base_img>
```

Comment:
Should add descriptions for these to the bullet list above.

Reply:
Unfortunately, I don't have sufficient knowledge about "show_headnode" to do this.

Reply:
* "uuid", UUID for the headnode
* "base_img", the os image that the headnode is running 


(see this [link](https://github.com/CCI-MOC/hil/blob/master/examples/hil.cfg#L29))

Reply:
Thanks